src/ParallelHashMap.h

```diff
@@ -255,9 +255,9 @@ void FixedHashMap<K,V>::insert(K key, V value)
 
   /* place element in linked list */
   *iter_node = new_node;
-
+  
   /* increment counter */
-#pragma omp atomic
+  #pragma omp atomic
```

Comment:
I was just fixing the whitespace in this PR. @geogunow fixes the indents in his current PR #265